elephant/test/test_statistics.py

```diff
@@ -335,5 +335,126 @@ def test_lv_raise_error(self):
         self.assertRaises(AttributeError, es.lv, 1)
         self.assertRaises(ValueError, es.lv, np.array([seq, seq]))
 
+
+class RateEstimationTestCase(unittest.TestCase):
+
+    def setUp(self):
+        # create a poisson spike train:
+        self.st_tr = (0, 20.0)  # seconds
+        self.st_dur = self.st_tr[1] - self.st_tr[0]  # seconds
+        self.st_margin = 5.0  # seconds
+        self.st_rate = 10.0  # Hertz
+
+        st_num_spikes = np.random.poisson(self.st_rate*(self.st_dur-2*self.st_margin))
+        spike_train = np.random.rand(st_num_spikes) * (self.st_dur-2*self.st_margin) + self.st_margin
+        spike_train.sort()
+        
+        # convert spike train into neo objects
+        self.spike_train = neo.SpikeTrain(spike_train*pq.s,
+                                          t_start=self.st_tr[0]*pq.s,
+                                          t_stop=self.st_tr[1]*pq.s)
+
+    def test_instantaneous_rate(self):
+        st = self.spike_train
+        sampling_period = 0.01*pq.s
+        inst_rate = es.instantaneous_rate(
+            st, sampling_period, 'TRI', 0.03*pq.s)
+        self.assertEquals(type(inst_rate), neo.core.AnalogSignalArray)
```

Comment:
This can be written as:
```
self.assertIsInstance(inst_rate, neo.core.AnalogSignalArray)
```
which gives a more useful error message in case of failure.